src/enum/ErrorOption.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorOption {
+  Clear = '',
```

Comment:
If there is no error, just set the state to `null` without creating an extra enum

Reply:
I reimplement that for null.